server/api/crud/model_monitoring/deployment.py

```diff
@@ -246,6 +247,73 @@ def deploy_model_monitoring_writer_application(
                 "writer_ready": ready,
             }
 
+    def apply_and_create_stream_trigger(
+        self, function: mlrun.runtimes.ServingRuntime, function_name: str = None
+    ) -> mlrun.runtimes.ServingRuntime:
+        """Adding stream source for the nuclio serving function. By default, the function has HTTP stream trigger along
+        with another supported stream source that can be either Kafka or V3IO, depends on the stream path schema that is
+        defined under mlrun.mlconf.model_endpoint_monitoring.store_prefixes. Note that if no valid stream path has been
+        provided then the function will have a single HTTP stream source.
+
+        :param function:                    The serving function object that will be applied with the stream trigger.
+        :param function_name:               The name of the function that be applied with the stream trigger,
+                                            None for model_monitoring_stream
+
+        :return: ServingRuntime object with stream trigger.
+        """
+
+        # Get the stream path from the configuration
+        stream_paths = server.api.crud.model_monitoring.get_stream_path(
+            project=self.project, function_name=function_name
+        )
+        for i, stream_path in enumerate(stream_paths):
+            if stream_path.startswith("kafka://"):
+                topic, brokers = mlrun.datastore.utils.parse_kafka_url(url=stream_path)
+                # Generate Kafka stream source
+                stream_source = mlrun.datastore.sources.KafkaSource(
+                    brokers=brokers,
+                    topics=[topic],
+                )
+                function = stream_source.add_nuclio_trigger(function)
+
+            if not mlrun.mlconf.is_ce_mode():
+                if stream_path.startswith("v3io://"):
+                    if "projects" in stream_path:
+                        stream_args = (
+                            config.model_endpoint_monitoring.application_stream_args
+                        )
+                        access_key = self.model_monitoring_access_key
+                        kwargs = {"access_key": self.model_monitoring_access_key}
+                    else:
+                        stream_args = (
+                            config.model_endpoint_monitoring.serving_stream_args
+                        )
+                        access_key = os.environ.get("V3IO_ACCESS_KEY")
+                        kwargs = {}
+                    if mlrun.mlconf.is_explicit_ack(version=resolve_nuclio_version()):
+                        kwargs["explicit_ack_mode"] = "explicitOnly"
+                        kwargs["worker_allocation_mode"] = "static"
+                    server.api.api.endpoints.functions.create_model_monitoring_stream(
```

Comment:
```suggestion
                    server.api.api.endpoints.nuclio.create_model_monitoring_stream(
```
for now just fix this, in the future we will move this function to `MonitoringDeployment` class